core/dbt/cli/flags.py

```diff
@@ -344,7 +342,8 @@ def add_fn(x):
 
         if k == "macro" and command == CliCommand.RUN_OPERATION:
             add_fn(v)
-        elif v in (None, False):
+        # None is Signletons, False is flyweight, only one instance of each.
```

Comment:
Nit: Singleton is misspelled.

Here's one way it could be rewritten slightly:

```suggestion
        # None is a Singleton, False is a Flyweight, only one instance of each.
```

Reply:
I blame it on copilot LOL